frontend/src/components/CreateQuestion/CreateQuestion.css

```diff
@@ -1,22 +1,4 @@
 .create-question-form {
     width: 80%;
     margin: 0 auto;
-}
-
-#createQuestionDescription {
-    height: 200px;
-}
-
-.react-tagsinput--focused {
-  border-color: #86b7fe;
-  box-shadow: 0 0 2px 3px rgba(194,219,254,255);
-}
-
-.react-tagsinput {
-  background-color: #fff;
-  border: 1px solid #ccc;
-  overflow: hidden;
-  padding-left: 5px;
-  padding-top: 5px;
-  border-radius: 5px;
-}
```

Comment:
Should not be deleted as it is styling the tags input to look more consistent with the other Bootstrapped components.
ie: Make the border blue upon clicking

Reply:
CSS file is actually not removed, but rather move to global CSS folder. Forgot to import it